stl/inc/valarray

```diff
@@ -60,6 +60,18 @@ class valarray { // store array with various indexing options
 private:

```

Comment:
(Pre-existing) friend declarations aren't accessible, so there's nothing here to make `private`:
```suggestion
public:
```